content/en/observability_pipelines/guide/send_splunk_logs_to_datadog.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Send Splunk Logs to Datadog
```

Comment:
I'm thinking about changing the title to "Dual-ship Splunk Logs to Datadog", so that it's more specific.